full/src/test/java/apoc/cypher/CypherExtendedTest.java

```diff
@@ -160,6 +160,19 @@ public void testRunFileWithResults() throws Exception {
                     assertEquals(false, r.hasNext());
                 });
     }
+
+    @Test
+    public void shouldNotFailWithTransactionErrorWithMapParallel2() {
+        // Flaky test. Sometimes it's green even without `db.beginTx()` modification
```

Comment:
it's not flaky anymore with the fix, is it?

Reply:
Of course, it was confusing. Changed.